### DISCUSSION TITLE
Remove pointless local network restriction from transport.py

### DISCUSSION THREAD

**Discussion by bcGITster** [mealie-recipes/mealie#3783] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

So I was trying to setup a process for me to consolidate recipes into a master meal recipe which I could view as a signel recipe page and have Mealie scrape that to simplify the process and what do I find, this garbage code the keeps you from accessing a recipe on your LAN.

Seriously, if someone got through your login, and guessed arbitray IP addresses to probe, all they're going to be able to do is guess a web server is running? This is the laziest brute force security solution looking for a problem that I've seen in a while. Can we please revert this out of the code? It's bad enough I can't have meals as a collection of recipies out of the box, but then when I try to make a work around to solve the problem I hit crap like this. If the guy who coded this as the solution worked for my shop he'd have been fired on the spot.

### Please provide a concise description of the feature that would resolve your issue.

Revert the restriction for accessing LAN resources.

### Please consider and list out some caveats or tradeoffs made in your design decision

Apparently someone with a login to mealie could check if an arbitrary IP on my lan has HTTP running? FFS.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
Hey man, I get that you are frustrated hitting that roadblock but I don't think your "colorful" language is needed here.

Mealie is build and maintained by a few passionate volunteers that don't get paid for their work so we are way less inclined to do anything if people try to bully us into doing it.

**Reply by bcGITster** [reply]
I'm sorry man. You're right. It's just been rough and I was at peak frustration. No excuse though. Please accept my apology.

**Comment by boc-the-git** [comment]
Kuchen has already given as nice a reply as you will get, so I'll close this.

**Reply by bcGITster** [reply]
Understood. As I said. I'm sorry for the outburst. It was out-of-line. Can someone explain to me what they were actually trying to solve with this fix? Maybe I can help with a re-write to address the security concern without the blanket LAN block.
